app/src/main/java/org/openobservatory/ooniprobe/adapter/TestsListAdapter.java

```diff
@@ -171,5 +201,17 @@ public boolean onMenuItemClick(MenuItem menuItem) {
         // Finally show the PopupMenu
         popup.show();
     }
+
+    private String getDate(long time) {
+        Calendar cal = Calendar.getInstance(Locale.ENGLISH);
+        cal.setTimeInMillis(time);
+        String date = DateFormat.format("yyyy-MM-dd HH:mm:ss", cal).toString();
+        return date;
+    }
+
+    public static int getStringIdentifier(Context context, String name) {
+        //TODO could cause a crash if a test name is not translated or not present in string file
+        return context.getResources().getIdentifier(name, "string", context.getPackageName());
```

Comment:
Perhaps we can factor this not-unlikely-throwing function that hurt me earlier in a common place?